src/i18n/i18n.translations.ts

```diff
@@ -25,8 +26,8 @@ export const es = {
     [namespaces.pages.loginScreen]: loginScreenEs,
     [namespaces.pages.menuDeleteAccount]: menuDeleteAccountEs,
     [namespaces.pages.menuReferral]: menuReferralEs,
-    [namespaces.pages.notifications]: {title: "Notificaciones"},
-    [namespaces.pages.pickupPoint]: {title: "Punto de recogida", address: "Dirección"},
+    [namespaces.pages.notifications]: { title: "Notificaciones" },
+    [namespaces.pages.pickupPoint]: { title: "Punto de recogida", address: "Dirección" },
```

Comment:
I guess you have added these attributes directly here because they are 1 or 2 only, but I think we should follow the same pattern as with the rest. For consistency with the rest, for reusability and also to have both texts in the same place, I think it would be better to create an i18n page with both English and Spanish texts.

Reply:
@braisgonzalez Sure, but I didn't originally add this, it shows on the diff because my IDE added spaces between the braces.